api.md

```diff
@@ -1906,9 +1949,20 @@ The `start_offset` and `end_offset` parameters should be specified differently d
 |`job_id`| (INTEGER)  TimescaleDB background job id created to implement this policy|
 
 
+#### Errors
+
+An error will be given if:
+
+- The `start_offset` or `end_offset` specify a range less than two buckets in size.
```

Comment:
Do I follow correctly that the policy requires 2 buckets while the refresh function only 1 bucket? (just checking)

Reply:
Yes, the reality is that you _can_ supply the correct values to update one full bucket to the manual refresh, but a policy is always going to execute at a time that almost never would get you an exact bucket width if you only updated one bucket of time.

I think this is a little confusing overall, but it does provide a little bit more flexibility to users in the end.

Reply:
Thank you for clarifying! It will be definitely confusing to users. However, changing the bevaviour is outside this PR.

@erimatnor May be provide a place where the acceptable bucket sizes are explicitly mentioned between the policy and the function. Otherwise, some users might remember only one of the limitations or think that one of them a typo.

Reply:
@k-rus I am not sure I understand what you are suggesting, in particular: "May be provide a place where the acceptable bucket sizes are explicitly mentioned between the policy and the function."

Reply:
FWIW, it is correct that the policy requires two buckets and refresh function only one, for the very reason @ryanbooz explains.

I think the refresh function is doing things correctly, and the policy is the awkward interface. This is, IMO, a _general_ problem with the background scheduling framework and policies in general. For instance, there is no way to control when the policy runs; one would likely prefer CRON-like scheduling over what is implemented now (there are also multiple issues for this). Perhaps allowing people to say "always refresh the latest bucket or something" is a better interface, but that also has issues since it isn't obvious what the bucket size is. 

I do think this might be a smaller problem that we think though. Policies that refresh less than two buckets don't make a whole lot of sense anyway, and I think it is much better to raise an error if people try to create them than silently accepting something that is semi-broken. So hopefully this is not the common case and a big problem in the end.

Reply:
It is common that users try to specify policy to refresh only one bucket, I've seen this number of times.
It is confusing/bad UX that the policy and refresh functions accept different refresh windows. So users might miss that the requirements are different between the policy and the function. Thus I suggest to mention explicitly that the refresh function and the cagg policy have different minimal requirement on the refresh window, e.g.,
> Note that the refresh window in the policy needs to be at least two buckets, while in the refresh function at least one bucket.